scene/gui/code_edit.h

```diff
@@ -185,6 +185,9 @@ class CodeEdit : public TextEdit {
 	bool code_hint_draw_below = true;
 	int code_hint_xpos = -0xFFFF;
 
+	/* Code region */
+	String _get_code_region_tag(String p_tag) const;
```

Comment:
Should be grouped under folding.

Could it also be made a `const String &p_tag`?